### ISSUE TITLE
[iOS Designer] Freeformsize doesn't allow to change xib's size

### ISSUE THREAD

**Issue by xamadev** [xamarin/xamarin-macios#6022]
### Steps to Reproduce

See post on GitHub: https://stackoverflow.com/questions/56025426/how-to-change-xibs-size-in-xamarin-ios-designer

I'm facing the issue on Visual Studio for Windows, haven't tested it on Mac. Another user tested it in Visual Studio for Windows/Mac and also faces the issue.

### Expected Behavior

Changing xib's size per drag on the Design Surface or per Layout tab.

### Actual Behavior

Changing xib's size with no described way possible even in xml.

### Environment

```
Microsoft Visual Studio Professional 2017 
Version 15.9.11
VisualStudio.15.Release/15.9.11+28307.586
Microsoft .NET Framework
Version 4.7.03190

Installierte Version: Professional

Allgemeine Azure-Tools   1.10
Bietet allgemeine Dienste für die Verwendung durch Azure Mobile Services und Microsoft Azure-Tools.

Application Insights-Tools für Visual Studio-Paket   8.14.20131.1
Application Insights-Tools für Visual Studio

ASP.NET and Web Tools 2017   15.9.04012.0
ASP.NET and Web Tools 2017

ASP.NET Core Razor Language Services   15.8.31590
Provides languages services for ASP.NET Core Razor.

ASP.NET Web Frameworks and Tools 2017   5.2.61435.0
For additional information, visit https://www.asp.net/

Azure App Service Tools v3.0.0   15.9.03024.0
Azure App Service Tools v3.0.0

C#-Tools   2.10.0-beta2-63501-03+b9fb1610c87cccc8ceb74a770dba261a58e39c4a
C#-Komponenten, die in der IDE verwendet werden. Je nach Projekttyp und Einstellungen kann eine andere Compilerversion verwendet werden.

Extensibility Message Bus   1.1.49 (remotes/origin/d15-8@ee674f3)
Provides common messaging-based MEF services for loosely coupled Visual Studio extension components communication and integration.

GitHub.VisualStudio   2.8.1.7307
A Visual Studio Extension that brings the GitHub Flow into Visual Studio.

JavaScript-Sprachdienst   2.0
JavaScript-Sprachdienst

JetBrains ReSharper Ultimate 2018.2.3   Build 182.0.20180912.70621
JetBrains ReSharper Ultimate package for Microsoft Visual Studio. For more information about ReSharper Ultimate, visit http://www.jetbrains.com/resharper. Copyright © 2019 JetBrains, Inc.

Microsoft Continuous Delivery Tools für Visual Studio   0.4
Die Konfiguration von Azure DevOps-Pipelines von der Visual Studio-IDE aus wird vereinfacht.

Microsoft JVM Debugger   1.0
Provides support for connecting the Visual Studio debugger to JDWP compatible Java Virtual Machines

Microsoft MI-Based Debugger   1.0
Provides support for connecting Visual Studio to MI compatible debuggers

Microsoft Visual Studio-Tools für Container   1.1
Jetzt können Sie Ihre ASP.NET Core-Anwendung in der Zielumgebung entwickeln, ausführen und überprüfen. Drücken Sie F5, um die Anwendung direkt in einem Container mit Debugfunktionen auszuführen, oder STRG+F5 zum Bearbeiten und Aktualisieren Ihrer Anwendung, ohne den Container erneut erstellen zu müssen.

Mono Debugging for Visual Studio   4.13.12-pre (9bc9548)
Support for debugging Mono processes with Visual Studio.

NuGet-Paket-Manager   4.6.0
NuGet-Paket-Manager in Visual Studio. Weitere Informationen zu NuGet finden Sie unter "http://docs.nuget.org/".

ProjectServicesPackage Extension   1.0
ProjectServicesPackage Visual Studio Extension Detailed Info

ResourcePackage-Erweiterung   1.0
Detaillierte Informationen zur ResourcePackage-Erweiterung von Visual Studio

ResourcePackage-Erweiterung   1.0
Detaillierte Informationen zur ResourcePackage-Erweiterung von Visual Studio

SQL Server Data Tools   15.1.61903.01040
Microsoft SQL Server Data Tools

TypeScript-Tools   15.9.20918.2001
TypeScript-Tools für Microsoft Visual Studio

Visual Basic-Tools   2.10.0-beta2-63501-03+b9fb1610c87cccc8ceb74a770dba261a58e39c4a
Visual Basic-Komponenten, die in der IDE verwendet werden. Je nach Projekttyp und Einstellungen kann eine andere Version des Compilers verwendet werden.

Visual F# Tools 10.2 für F# 4.5   15.8.0.0.  Commit Hash: 6e26c5bacc8c4201e962f5bdde0a177f82f88691.
Microsoft Visual F# Tools 10.2 für F# 4.5

Visual Studio-Tools für Container   1.0
Visual Studio-Tools für Container

Visual Studio Code-Debugadapter-Hostpaket   1.0
Interopebene zum Hosten der Visual Studio Code-Adapter in Visual Studio

VisualStudio.Mac   1.0
Mac Extension for Visual Studio

VisualSVN   6.3.1
Integration with Subversion version control. For more information about VisualSVN, see the VisualSVN website at https://www.visualsvn.com
Copyright © 2019 VisualSVN Software Ltd. All rights reserved.

Xamarin   4.12.3.83 (d15-9@23fa80172)
Visual Studio-Erweiterung, um Entwicklung für Xamarin.iOS und Xamarin.Android zu ermöglichen.

Xamarin Designer   4.16.30 (e0af99a34)
Visual Studio extension to enable Xamarin Designer tools in Visual Studio.

Xamarin Templates   1.1.128 (6f5ebb2)
Templates for building iOS, Android, and Windows apps with Xamarin and Xamarin.Forms.

Xamarin.Android SDK   9.1.7.0 (HEAD/ba9da7a76)
Xamarin.Android Reference Assemblies and MSBuild support.

Xamarin.iOS and Xamarin.Mac SDK   12.4.0.64 (9c8d8e0)
Xamarin.iOS and Xamarin.Mac Reference Assemblies and MSBuild support.
```

### Example Project

See repo: https://github.com/xamadev/CustomRendererUIView

> VS bug [#891380](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/891380)

**Comment by xamadev** [comment]
Workaround found. Look at marked answer's comments on GitHub.

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-9] Comment by **xamarinc**



Fixed in version 16.3.99.121 (master)



Author: Denis Kandrov  
 Commit: d34bac4890b48a1a8a1fa4e88cf419d24f3efe4d (xamarin/designer)
